technic/machines/HV/nuclear_reactor.lua

```diff
@@ -209,9 +226,21 @@ local function reactor_structure_badness(pos)
 	return (25 - water_layer) + (96 - lead_layer) + (216 - blast_layer)
 end
 
+local mcl_expl_info = {
+   drop_chance = 1.0,
+   max_blast_resistance = 10,
+   sound = true,
+   particles = true,
+   fire = true,
+   griefing = true,
+   grief_protected = true,
+}
```

Comment:
Just indentation / style
```suggestion
local mcl_expl_info = {
	drop_chance = 1.0,
	max_blast_resistance = 10,
	sound = true,
	particles = true,
	fire = true,
	griefing = true,
	grief_protected = true,
}
```